Update the description for DataGrid - Adaptability Overview (#82)
DevExpress/devextreme-demos: JSDemos/Demos/DataGrid/GridAdaptabilityOverview/description.md

> **DataGrid** automatically adapts its layout to screens of different widths. In this demo, you can switch between horizontal and vertical screen orientations. If columns do not fit the chosen orientation, the widget hides them one by one, starting with the rightmost column. Information from the hidden columns is still available in adaptive detail rows.

Reviewer: ```suggestion
**DataGrid** automatically adapts its layout to screens with different sizes. In this demo, you can switch between horizontal and vertical screen orientations. If columns do not fit the selected orientation, the widget hides them one by one, starting with the rightmost column. Information from the hidden columns is still available in adaptive detail rows.
```

> **DataGrid** automatically adapts its layout to screens with different sizes. In this demo, you can switch between horizontal and vertical screen orientations. If columns do not fit the selected orientation, the widget hides them one by one, starting with the rightmost column. Information from the hidden columns is still available in adaptive detail rows.

Reviewer 2: "Information from the hidden columns is still available in **adaptive detail rows**."
This sentence left me wondering what these 'adaptive detail rows' are.  The sentence from the previous version of  the demo description explains this concept better: "To view values of hidden columns, click the ellipsis button in the last column of the required row.", doesn't it?

> To enable this behavior in your application, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#columnHidingEnabled) option to **true**.

Reviewer: ```suggestion
To enable this feature, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#columnHidingEnabled) option to **true**.
```

> **DataGrid** automatically adapts its layout to screens of different widths. Use buttons in the top right corner to switch between the mobile and tablet screens. If columns do not fit the chosen screen type, the widget hides them one by one from right to left. 

Reviewer 3: ```suggestion
**DataGrid** automatically adapts its layout to screens of different widths. In this demo, you can switch between horizontal and vertical screen orientations. If columns do not fit the chosen orientation, the widget hides them one by one, starting with the rightmost column. Information from the hidden columns is still available in adaptive detail rows.
```

To enable this behavior in your application, set the [columnHidingEnabled](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#columnHidingEnabled) option to **true**.

> **DataGrid** displays hidden columns' data in the adaptive row as [Form](/Documentation/Guide/Widgets/Form/Overview/) items. You can use the [onAdaptiveDetailRowPreparing](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onAdaptiveDetailRowPreparing) handler to access this data. Refer to the handler's description to see the limitations.

Reviewer 3: The paragraph looks unnecessary. This handler does allow you to customize the adaptive detail row, but it seems that only few customers really need that. And the demo doesn't show this handler.

> Refer to the handler's description to see the limitations.

I think we mention limitations too often, it makes us look bad. We shouldn't do that unless the limitations are serious and easy to encounter.
